go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/ava-labs/teleporter
 go 1.21.12
 
 require (
-	github.com/ava-labs/avalanchego v1.11.9-0.20240703222923-9d71967ccbba
+	github.com/ava-labs/avalanchego v1.11.10-0.20240715192312-f62135695960
```

Comment:
In my local environment, this change causes:

```
$ go mod tidy
go: github.com/ava-labs/coreth@v0.13.6-rc.1.0.20240702201359-ba2ce5367874: invalid version: unknown revision ba2ce5367874
```

it's not clear to me why running `go mod tidy` in the CI environment actually suggests _adding_ this _unknown_ revision into the go.mod file. (see the CI result)